llvm/unittests/ProfileData/CoverageMappingTest.cpp

```diff
@@ -64,15 +64,18 @@ namespace {
 struct OutputFunctionCoverageData {
   StringRef Name;
   uint64_t Hash;
+  std::vector<std::string> FilenamesStorage;
   std::vector<StringRef> Filenames;
```

Comment:
Why not store the strings directly in Filenames rather than having a separate storage?

```suggestion
  std::vector<std::string> Filenames;
```